moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -100,6 +101,7 @@ const ProjectNotes = (props) => {
   const currentPhaseId =
     props.currentPhaseId ??
     props.data?.moped_project[0]?.moped_proj_phases[0]?.moped_phase.phase_id;
+  const snackbarHandle = props.snackbarHandle;
```

Comment:
this component ought to be refactored to use destructured props but i think that was out of scope here